### ISSUE TITLE
ThingWeb uses TD names to build URI paths

### ISSUE THREAD

**Issue by mkovatsc** [thingweb/thingweb#14]
Names are primarily human readable and not unique.
It would be better to use the TD hrefs values.

**Comment by danielpeintner** [comment]
Agree. Fixing it now before F2F seems dangerous tough. Should do that after the PlugFest.
